Improve the view more and see our blog buttons

Susmita0202 opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

## Description

added more hover effects on view more and see our blog buttons.
 issue no #420

## Motivation and Context

<!--- Why is this change required? What problem does it solve? -->
<!--- If it fixes an open issue, please link to the issue here. -->

## How Has This Been Tested?

<!--- Please describe in detail how you tested your changes. -->
<!--- Include details of your testing environment, the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->

## Screenshots (if appropriate):
![Screenshot 2025-03-15 101230](https://github.com/user-attachments/assets/fa2fcde2-cd1d-4a2c-8384-1c142b219662)
![Screenshot 2025-03-15 101250](https://github.com/user-attachments/assets/786dace9-794a-479c-b5c2-05633f9a44ef)
![Screenshot 2025-03-15 101322](https://github.com/user-attachments/assets/d3eb7ff7-5b71-4ffa-8ee6-f6da3fdbb85f)
![Screenshot 2025-03-15 101406](https://github.com/user-attachments/assets/2eaeedf5-6e7b-405d-9f3b-54080d1b25b7)


## Types of changes

<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

saismrutiranjan18 commented:
> <!--- Provide a general summary of your changes in the Title above -->
> 
> ## Description
> 
> added more hover effects on view more and see our blog buttons.
>  issue no #420
> 
> ## Motivation and Context
> 
> <!--- Why is this change required? What problem does it solve? -->
> <!--- If it fixes an open issue, please link to the issue here. -->
> 
> ## How Has This Been Tested?
> 
> <!--- Please describe in detail how you tested your changes. -->
> <!--- Include details of your testing environment, the tests you ran to -->
> <!--- see how your change affects other areas of the code, etc. -->
> 
> ## Screenshots (if appropriate):
> 
> ## Types of changes
> 
> <!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
> 
> - [ ] Bug fix (non-breaking change which fixes an issue)
> - [ ] New feature (non-breaking change which adds functionality)
> - [ ] Breaking change (fix or feature that would cause existing functionality to change)
> 
> ## Checklist:
> 
> <!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
> <!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
> 
> - [ ] My code follows the code style of this project.
> - [ ] My change requires a change to the documentation.
> - [ ] I have updated the documentation accordingly.

Will you provide screenshot? @Susmita0202

Susmita0202 commented:
Yes ofcourse

On Sat, Mar 15, 2025, 10:05 AM Sai Smruti Ranjan Das <
***@***.***> wrote:

> Description
>
> added more hover effects on view more and see our blog buttons.
> issue no #420
> <https://github.com/saismrutiranjan18/Tiffin_Fusion/issues/420>
> Motivation and Context How Has This Been Tested? Screenshots (if
> appropriate): Types of changes
>
>    - Bug fix (non-breaking change which fixes an issue)
>    - New feature (non-breaking change which adds functionality)
>    - Breaking change (fix or feature that would cause existing
>    functionality to change)
>
> Checklist:
>
>    - My code follows the code style of this project.
>    - My change requires a change to the documentation.
>    - I have updated the documentation accordingly.
>
> Will you provide screenshot? @Susmita0202 <https://github.com/Susmita0202>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/saismrutiranjan18/Tiffin_Fusion/pull/421#issuecomment-2726203401>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A7SRYOBQ3KHLDJQDZL7Y43T2UOUYDAVCNFSM6AAAAABZCAZ2QCVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOMRWGIYDGNBQGE>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
> [image: saismrutiranjan18]*saismrutiranjan18* left a comment
> (saismrutiranjan18/Tiffin_Fusion#421)
> <https://github.com/saismrutiranjan18/Tiffin_Fusion/pull/421#issuecomment-2726203401>
>
> Description
>
> added more hover effects on view more and see our blog buttons.
> issue no #420
> <https://github.com/saismrutiranjan18/Tiffin_Fusion/issues/420>
> Motivation and Context How Has This Been Tested? Screenshots (if
> appropriate): Types of changes
>
>    - Bug fix (non-breaking change which fixes an issue)
>    - New feature (non-breaking change which adds functionality)
>    - Breaking change (fix or feature that would cause existing
>    functionality to change)
>
> Checklist:
>
>    - My code follows the code style of this project.
>    - My change requires a change to the documentation.
>    - I have updated the documentation accordingly.
>
> Will you provide screenshot? @Susmita0202 <https://github.com/Susmita0202>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/saismrutiranjan18/Tiffin_Fusion/pull/421#issuecomment-2726203401>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A7SRYOBQ3KHLDJQDZL7Y43T2UOUYDAVCNFSM6AAAAABZCAZ2QCVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOMRWGIYDGNBQGE>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
